src/Domain.LinnApps/Parts/PartService.cs

```diff
@@ -345,17 +345,35 @@ public Part CreateFromSource(int sourceId, int createdBy, IEnumerable<PartDataSh
                                       { "address", bcc2.EmailAddress }
                                   }
                           };
-            this.emailService.SendEmail(
-                to.EmailAddress,
-                to.User.Name,
-                cc,
-                null,
-                "stores@linn.co.uk",
-                "Parts Utility",
-                $"New Source Sheet Created - {source.PartNumber}",
-                $"Click here to view: https://app.linn.co.uk/parts/sources/{source.Id}",
-                null,
-                null);
+            if (source.MechanicalOrElectrical == "E")
+            {
+                this.emailService.SendEmail(
+                    to.EmailAddress,
+                    to.User.Name,
+                    cc,
+                    null,
+                    "stores@linn.co.uk",
+                    "Parts Utility",
+                    $"New Source Sheet Created - {source.PartNumber}",
+                    $"Click here to view: https://app.linn.co.uk/parts/sources/{source.Id}",
+                    null,
+                    null);
+            }
+            else
+            {
+                this.emailService.SendEmail(
+                    "mechanicalsourcingsheet@linn.co.uk",
```

Comment:
Is this a real email address / inbox that makes it through to real people? if so avoid leaking it into code (especially in public repos like this one) since spam bots will be scouring the web for them. Could stick this in a config.env or something?

Reply:
it also might be worth finding out who has access to this inbox if it is a real invoice? If its the same group of people that I've hardcoded on lines 331-333 then there's maybe no need for the if/else at all? If the recipients are the same then the if/else branches are functionally identical